subprojects/packagefiles/imgui/meson.build

```diff
@@ -53,10 +53,10 @@ if metal_dep.found()
     sources += 'backends/imgui_impl_metal.mm'
     dependencies += metal_dep
 endif
-libdl_dep = cpp.find_library('dl', required: get_option('opengl'))
-if libdl_dep.found()
+libgl_dep = dependency('GL', required: get_option('opengl'))
```

Comment:
why is GL capitalized?

Reply:
It is capitalized in the [meson manual](https://mesonbuild.com/Dependencies.html#gl), so I also do that. This way it compiles on windows and on linux, so i did not try it in small letters.

Reply:
You should be able to lower-case it (see e.g. Boost, CUDA, AppleFrameworks in the manual)